src/compiler.ts

```diff
@@ -355,16 +355,26 @@ export const enum RuntimeFeatures {
   setArgumentsLength = 1 << 6
 }
 
+/** Imported default names of compiler-generated elements. */
+export namespace ImportNames {
+  /** Name of the default namespace */
+  export const DefaultNamespace = "env";
+  /** Name of the memory instance, if imported. */
+  export const Memory = "memory";
+  /** Name of the table instance, if imported. */
+  export const Table = "table";
+}
+
 /** Exported names of compiler-generated elements. */
 export namespace ExportNames {
+  /** Name of the memory instance, if exported. */
+  export const Memory = "memory";
+  /** Name of the table instance, if exported. */
+  export const Table = "table";
```

Comment:
What consideration were there to make these uppercase? I think in all other places we are just using lowercase names.

Reply:
I don't have a strict preference. But I think I should normalize everything to CamelCase-style all enums / namespaced constants similar to `UnaryOp.Eq32` and others